src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
I'm unsure about gamepad controls in the core. Maybe too prescriptive? Trying to do a use case where I want the right joystick to snap the rotation in like 30-degree increments, so I feel like immediately gamepad controls should be left to the developer because each app is so specific. Like developer might not want DPad to do velocity as well.

But we'd definitely publish it separately if we excluded it here.

@donmccurdy @dmarcos